pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -772,6 +774,8 @@ func TestReconciler(t *testing.T) {
 				*basePodWrapper.
 					Clone().
 					Label(constants.ManagedByKueueLabel, "true").
+					Label(controllerconsts.PodSetLabel, "dc85db45").
```

Comment:
Can you define these as constants and reuse that here?

Reply:
I'm not sure this is the best course of action, because:
1. this string is used 105 times in the file already: https://github.com/kubernetes-sigs/kueue/blob/b6222fa0ec020bad0d93be9acdd0305348c4935a/pkg/controller/jobs/pod/pod_controller_test.go#L601, and so it would mix a feature with extended cleanup
2. if I introduce a const like `podSetNameForTest` at the top of the file it could suggest it should be used in all tests, but it is derived from the Pods actually, it was just copy-pasted many times. 

So, I don't see this would be very clean, but let me know your thoughts.

Reply:
Would you mind if I do a cleanup?

I agree with you that this should not be in scope for this feature then. I didn't see how many occurences this has.

Reply:
Sure, feel free to follow up! maybe instead of hard-coding a const we could compute the hash in tests, but not sure at the moment.